libwacom/libwacom-database.c

```diff
@@ -295,32 +419,65 @@ libwacom_parse_stylus_keyfile(WacomDeviceDatabase *db, const char *path)
 
 	groups = g_key_file_get_groups (keyfile, NULL);
 	for (i = 0; groups[i]; i++) {
-		WacomStylus *stylus;
-		WacomStylusId id;
+		WacomStylus *stylus, *other;
```

Comment:
Is there a word like `aliasee`? Or maybe `source_stylus`? Or some other word that tells us immediately that this is the one we're aliasing to (from?).